.github/workflows/run_periodic_tests.yml

```
@@ -153,3 +153,47 @@ jobs:
           eval "$(pyenv init -)"
           pyenv activate pybamm-${{ matrix.python-version }}
           pyenv uninstall -f $( python --version )
+
+  test_install_odes:
+    needs: style
+    runs-on: macos-latest
```

```suggestion
    runs-on: ${{ matrix.os }}
```

Doesn't require the `style` job to pass since no extra code will be checked out here